R/template.R

```diff
@@ -759,6 +759,14 @@ yardstick_eval_select <- function(expr,
                                   error_call = caller_env()) {
   check_dots_empty(call = error_call)
 
+  if (!quo_is_missing(expr) && inherits(quo_get_expr(expr), "name")) {
+    expr_name <- as_name(expr)
```

Comment:
not a big deal, but if you refactor out this to the line before then you can say 

```r
  set_known_selection(expr_name, out)
```

instead of

```r
  set_known_selection(as_name(expr), out)
```

Reply:
Yes, but `as_name()` expr errors if either of the above conditions aren't true.